docker/deploy.sh

```diff
@@ -196,8 +211,11 @@ done
 # place the domain name into the deployment file, in a backup
 docker_file_path=$docker_path/$docker_type
 docker_file=$docker_file_path/docker-compose.yml
+gw_docker_file=$docker_file_path/gw-docker-compose.yml
 docker_backup_file=$docker_file_path/docker-compose.yml.bak
+gw_docker_backup_file=$docker_file_path/gw_docker-compose.yml.bak
```

Comment:
If `$docker_type` is set to "compose", this should, in theory, result in a path like `/scim-examples/docker/compose/gw-docker-compose.yml` but there is no `gw-docker-compose.yml` file in `/scim-examples/docker/compose`. I suspect it's just a matter of creating copies of those yaml files in /compose? (maybe there's another way we can do this without creating duplicate files?)

Reply:
I have an idea for how to handle this situation cleanly.

Reply:
NICE! 🤩